db/db/migrations/20230307094014_orders_item.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE orders_item (
```

Comment:
```suggestion
CREATE TABLE order_items (
```
테이블 안에 들어가는 데이터를 생각해보면 orders_item 보다 order_items 가 더 정확한 네이밍입니다!